components/rhs_header_post/index.ts

```diff
@@ -40,6 +41,8 @@ function mapStateToProps(state: GlobalState, {rootPostId}: OwnProps) {
 
     const collapsedThreads = isCollapsedThreadsEnabled(state);
     const root = getPost(state, rootPostId);
+    const currentUserId = getCurrentUserId(state) as string;
```

Comment:
why is `string` casted onto `currentUserId`? `getCurrentUserId` should return a string only.